ttnn/cpp/ttnn/operations/conv/conv2d/prepare_conv2d_weights.cpp

```diff
@@ -377,6 +380,7 @@ ttnn::Tensor prepare_conv_weights(
         opt_conv_op_block_config.act_block_w_ntiles,
         opt_conv_op_block_config.out_subblock_w_ntiles,
         parallel_config,
+        parallel_config,
```

Comment:
is it OK pass the same thing here twice?

Reply:
No. I will fix it.